test_flake8_graphql.py

```diff
@@ -60,3 +101,31 @@ def my_function():
         '           ^',
         '5:             name',
     ]
+
+
+def test_GQL100_custom_identifer_fail(flake8dir):
+    flake8dir.make_example_py("""
+    class GQL(object):
```

Comment:
I actually opted to test import/class/function separately in order to catch possible regressions. I suppose here would make sense to have `GQL` imported since they are already covered though.